rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/models/BulkOperationResultsRepresentationModel.java

```diff
@@ -0,0 +1,47 @@
+package pro.taskana.task.rest.models;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import org.springframework.hateoas.RepresentationModel;
+import pro.taskana.common.api.exceptions.ErrorCode;
+
+/** EntityModel class for {@link pro.taskana.common.api.BulkOperationResults}. */
+public class BulkOperationResultsRepresentationModel
+    extends RepresentationModel<BulkOperationResultsRepresentationModel> {
+
+  /** Map of keys to the stored information. */
+  protected Map<String, ErrorCode> tasksWithErrors = new HashMap<>();
+
+  public Map<String, ErrorCode> getTasksWithErrors() {
+    return tasksWithErrors;
+  }
+
+  public void setTasksWithErrors(Map<String, ErrorCode> tasksWithErrors) {
+    this.tasksWithErrors = tasksWithErrors;
+  }
+
+  protected boolean canEqual(Object other) {
+    return (other instanceof BulkOperationResultsRepresentationModel);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(tasksWithErrors);
+  }
+
+  @Override
+  public boolean equals(Object obj) {
+    if (this == obj) {
+      return true;
+    }
+    if (!(obj instanceof BulkOperationResultsRepresentationModel)) {
```

Comment:
Why do you check "instanceof" twice?

Reply:
I will delete the function (see above comment)